src/lib/components/FileUploader/utils.js

```diff
@@ -9,20 +10,33 @@ import React from 'react';
 
 import _isNumber from 'lodash/isNumber';
 
-export function humanReadableBytes(bytes) {
+export function humanReadableBytes(bytes, decimalDisplay = false) {
   if (_isNumber(bytes)) {
-    const kiloBytes = 1000;
-    const megaBytes = 1000 * kiloBytes;
-    const gigaBytes = 1000 * megaBytes;
+    const base = decimalDisplay ? 1000 : 1024;
+    const kiloBytes = base;
+    const megaBytes = base * kiloBytes;
+    const gigaBytes = base * megaBytes;
 
     if (bytes < kiloBytes) {
       return <>{bytes} bytes</>;
     } else if (bytes < megaBytes) {
-      return <>{(bytes / kiloBytes).toFixed(2)} Kb</>;
```

Comment:
i think the lower-case "b" was a bit misleading, as AFAIK this is sometimes used to denote "bits" instead of "bytes" (c.f. [here](https://www.techtarget.com/searchnetworking/definition/Mbps))